cli/changelog/0.52.1.md

```diff
@@ -0,0 +1,4 @@
+### Fixes
+
+- Make MongoDB connector produce valid schemas when introspected (#1162)
+- Fix a bug in the rendering of GraphQL schemas where the interface description would be printed after its name (#1166)
```

Comment:
Don't think this bug would ever have manifested in the CLI - probably doesn't need to be in the changelog

Reply:
Hmm, the other one would also only show up in `dev`, so do we need this release at all?